### DISCUSSION TITLE
not scrolling to find element with iOS 16.0 simulator

### DISCUSSION THREAD

**Discussion by bbrown62450** [appium/appium#17707] [Q&A]
Has anyone else experienced their tests not scrolling to find an element with iOS 16.0 simulator. If I scroll manually down to the element while the tests are running, the element is found. I switched back to my 15.x simulator and scrolling worked as expected.

**Comment by jlipps** [comment]
I guess maybe Apple changed the behaviour when element interaction is requested. Appium itself does not attempt to scroll to elements when you interact with them. So if there's a breaking change here it would likely be on the Apple side. I would recommend submitting a bug report to them.
